### PR TITLE
New thumbnails: FACEIT & netcup

### PR DESCRIPTION
**Pull request by r00tusrDE**
Added new thumbnail for FACEIT & netcup

### CODE REVIEW

File: app/src/main/res/drawable/thumb_faceit.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="900dp"
+    android:height="900dp"
+    android:viewportWidth="900"
+    android:viewportHeight="900">
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_faceit.xml]: Please decrease size to at most 128x128

File: app/src/main/res/drawable/thumb_netcup.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="454dp"
+    android:height="454dp"
+    android:viewportWidth="454"
+    android:viewportHeight="454">
```
Comment (RichyHBM) [inline app/src/main/res/drawable/thumb_netcup.xml]: As above, please decrease size to at most 128x128

File: app/src/main/res/drawable/thumb_faceit.xml (hunk 1 above)
Comment (r00tusrDE) [reply app/src/main/res/drawable/thumb_faceit.xml]: Changed thumbnail size to 128x128 in ca70d29

File: app/src/main/res/drawable/thumb_netcup.xml (hunk 2 above)
Comment (r00tusrDE) [reply app/src/main/res/drawable/thumb_netcup.xml]: Changed thumbnail size to 128x128 in ca70d29

**Pull request merged**